src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1796,51 +1800,57 @@ private void CheckForSolutionFile()
 				.FirstOrDefault()?.ItemPath;
 		}
 
+		private void GetDesktopIniFileData()
+		{
+			var path = Path.Combine(WorkingDirectory, "desktop.ini");
+			DesktopIni = WindowsIniService.GetData(path);
+		}
+
 		public void CheckForBackgroundImage()
 		{
-			var iniPath = Path.Combine(WorkingDirectory, "desktop.ini");
-			if (!File.Exists(iniPath))
+			var filesAppSection = DesktopIni.FirstOrDefault(x => x.SectionName == "FilesApp");
+			if (filesAppSection is null)
 				return;
 
-			// Read data
-			var lines = File.ReadLines(iniPath);
-			var keys = lines.Select(line => line.Split('='))
-								.Where(parts => parts.Length == 2)
-								.ToDictionary(parts => parts[0].Trim(), parts => parts[1].Trim());
-
 			// Image source
-			if (folderSettings.LayoutMode is not FolderLayoutModes.ColumnView && keys.TryGetValue("Files_BackgroundImage", out var backgroundImage))
+			var backgroundImage = filesAppSection.Parameters?.FirstOrDefault(x => x.Key == "Files_BackgroundImage").Value;
+			if (string.IsNullOrEmpty(backgroundImage) is true)
+			{
+				return;
+			}
```

Comment:
Brackets can be removed

Reply:
for the future reference, brackets being removed even though theres `else` is not what our guideline suggests.